travis/hibench.conf

```diff
@@ -28,6 +32,14 @@ hibench.configure.dir		${hibench.home}/conf
 # default hibench HDFS root
 hibench.hdfs.data.dir		${hibench.hdfs.master}/HiBench
 
+# path of hibench jars
+hibench.hibench.datatool.dir	          ${hibench.home}/autogen/target/autogen-8.0-SNAPSHOT-jar-with-dependencies.jar
+hibench.common.jar                      ${hibench.home}/common/target/hibench-common-8.0-SNAPSHOT-jar-with-dependencies.jar
+hibench.sparkbench.jar                  ${hibench.home}/sparkbench/assembly/target/sparkbench-assembly-8.0-SNAPSHOT-dist.jar
+hibench.streambench.stormbench.jar      ${hibench.home}/stormbench/streaming/target/stormbench-streaming-8.0-SNAPSHOT.jar
+hibench.streambench.gearpump.jar        ${hibench.home}/gearpumpbench/streaming/target/gearpumpbench-streaming-8.0-SNAPSHOT-jar-with-dependencies.jar
+hibench.streambench.flinkbench.jar      ${hibench.home}/flinkbench/streaming/target/flinkbench-streaming-8.0-SNAPSHOT-jar-with-dependencies.jar
+
```

Comment:
I believe we already have these lines in hibench.conf, remove them?

Reply:
@gczsjdy  it's done.